admin/class-yoast-notification-center.php

```diff
@@ -397,17 +402,21 @@ public function remove_notification( Yoast_Notification $notification, $resolve
 
 		$index = false;
 
+		// ID of the user to show the notification for, defaults to current user id.
+		$user_id                = $notification->get_user_id();
+		$notifications_for_user = $this->get_notifications_for_user( $user_id );
```

Comment:
I would rename this to `$notifications`. Because it is an array of notifications.